engine/src/main/java/org/hibernate/validator/HibernateValidatorBootstrapConfiguration.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator;
+
+import javax.validation.BootstrapConfiguration;
+
+/**
+ * Contains Hibernate Validator specific bootstrap configurations.
+ *
+ * @author Marko Bekhta
+ * @since 6.1
+ */
+public interface HibernateValidatorBootstrapConfiguration extends BootstrapConfiguration {
+	/**
+	 * Class name of the {@link org.hibernate.validator.cfg.scriptengine.ScriptEvaluatorFactory} implementation or
+	 * {@code null} if none is specified.
+	 *
+	 * @return script evaluator factory class name or {@code null}
+	 *
+	 * @since 6.1
+	 */
+	default String getScriptEvaluatorFactoryClassName() {
```

Comment:
I don't think we should add this method. One can get the configured value via `BootstrapConfiguration#getProperties()`. And `BootstrapConfiguration` is not meant to expose the implementation-specific class names, instead there should be `HibernateValidatorConfiguration#getDefaultScriptEvaluatorFactory()` which returns an _instance_.